composer.json

```
@@ -7,8 +7,8 @@
     "php": ">=5.6",
     "composer/installers": "^v1.12.0 || ^2.2",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7 || ^1.0",
-    "wp-coding-standards/wpcs": "dev-develop#fca9d9e as 2.3.0",
-    "automattic/vipwpcs": "2.3.3",
+    "wp-coding-standards/wpcs": "dev-develop#bb792cb as 2.3.0",
```

```suggestion
    "wp-coding-standards/wpcs": "3.0.0",
```

The commit hash thing was just a workaround we needed before 3.0 was released.

---

Make sense. Update WPCS to 3.0 and `automattic/vipwpcs` to last commit hash for development branch.